app/src/main/java/mobile/programming/musicapp/dto/Instrument.kt

```diff
@@ -1,11 +1,8 @@
 package mobile.programming.musicapp.dto
 
-
 data class Instrument(var id: Int, var name: String, var type: String) {
 
     override fun toString(): String{
-
-        return (id.toString() + " " + name + " " + type)
+        return "$id $name $type"
```

Comment:
Modified your return output by replacing what you had with a string template, which can be found on https://kotlinlang.org/docs/reference/coding-conventions.html#string-templates

Reply:
Good suggestion, though this was covered in other reviews.

Reply:
I've changed the string template.